dali/pipeline/operators/reader/loader/video_loader.h

```diff
@@ -104,13 +104,16 @@ class VideoLoader : public Loader<GPUBackend, SequenceWrapper> {
     const std::vector<std::string>& filenames)
     : Loader<GPUBackend, SequenceWrapper>(spec),
       count_(spec.GetArgument<int>("count")),
+      step_(spec.GetArgument<int>("step")),
       height_(0),
       width_(0),
       filenames_(filenames),
       // TODO(spanev) handle device_id != 0
       device_id_(0),
       codec_id_(0),
       done_(false) {
+      if (step_ == -1)
+        step_ = count_;
```

Comment:
It would be nice to also handle the cases of `step<-1`

Reply:
Done